.github/workflows/cron_e2e.yaml

```diff
@@ -62,8 +64,8 @@ jobs:
           ./konstraint create -o e2e-resources test
 
       - name: create kind cluster
-        run: kind create cluster
-      
+        run: kind create cluster --image=kindest/node:v1.24.0
```

Comment:
I think another alternative would be to disable GK from trying to create PSP resources, via `--set psp.enabled=false` in the Helm install command. Either will unblock us in the short term though, up to you.

https://github.com/open-policy-agent/gatekeeper/blob/8a3e2159a0e8be138d46c4a199829a5b8d03d5d5/charts/gatekeeper/values.yaml#L235

Reply:
Oh! I didn't even realize that was configurable. That would probably be better, I'll give it a shot.

Reply:
Worked! 🎉